proposals/20200506-artifacts-scope-part-1.md

```diff
@@ -0,0 +1,114 @@
+# Falco Artifacts Scope - Part 1
+
+The **Falco Artifact Scope** proposal is divided in two parts:
+1. the Part 1 - *this document*: to “capture” the state of recent changes
+2. the [Part 2](./20200506-artifacts-scope-part-2.md): the intended state moving forward
+
+## Summary 
+
+As a project we would like to support the following artifacts.
+
+Everything else will be moved to [contrib](https://github.com/falcosecurity/contrib).
+
+As a project we will build, change, rename, and move files, documents, scripts, configurations according to this new state of the world.
+
+Inspired from [#1114](https://github.com/falcosecurity/falco/issues/1114) and many of the weekly community calls.
+
+## Terms
+
+**falco** 
+
+*Standalone Falco binary*
+
+**driver**
+
+*System call provider from the Linux kernel. Either (`bpf`, `module`, or `ptrace`)*
+
+**falco-driver-loader**
+
+*The bash script found [here](https://github.com/falcosecurity/falco/blob/master/scripts/falco-driver-loader) that tries to compile else download the driver (kernel module or eBPF probe).*
+
+**package**
+
+*An installable artifact that is operating system specific. All packages MUST be hosted on bintray.*
+
+**image**
+
+*OCI compliant container image hosted on dockerhub with tags for every release.*
+ 
+
+# Packages
+
+List of currently official packages (for x86 64bits only):
+
+- `falco-x.y.z-x86_64.deb` for debian like systems, it installs the kernel module by default
```

Comment:
This is currently correct, our package for deb is called `falco-x.y.z-x86_64.deb` 
- however this is not consistent against the [debian repository format ](https://wiki.debian.org/DebianRepository/Format?action=show&redirect=RepositoryFormat#Filename) where the architecture for x86_64 is referred as `amd64` unlikely how it is for rpm packages

Reply:
To evaluate for the Part 2